WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignIntroView.swift

```diff
@@ -0,0 +1,120 @@
+import SwiftUI
+
+/// View to display the introduction to the Blaze campaign
+///
+struct BlazeCampaignIntroView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Image(uiImage: .blaze)
+                        .frame(width: 100, height: 100)
+                        .clipShape(Circle())
+                        .background(
+                            Circle()
+                                .fill(Color(.withColorStudio(.orange, shade: .shade40)).opacity(0.08))
+                        )
+                        .padding(.bottom, Layout.elementVerticalSpacing)
+
+
+                    Text(Localization.title)
+                        .largeTitleStyle()
+                        .padding(.bottom, Layout.elementVerticalSpacing)
+
+                    BulletPointView(text: Localization.descriptionPoint1)
+                    BulletPointView(text: Localization.descriptionPoint2)
+                    BulletPointView(text: Localization.descriptionPoint3)
+                    BulletPointView(text: Localization.descriptionPoint4)
+                }
+                .padding(Layout.contentPadding)
+            }
+            .safeAreaInset(edge: .bottom) {
+                VStack {
+                    Divider()
+                        .frame(height: Layout.dividerHeight)
+                        .foregroundColor(Color(.separator))
+                    Button(Localization.startBlazeCampaign) {
+                        // todo
+                    }
+                    .buttonStyle(PrimaryButtonStyle())
+                    .padding(16)
+                }
```

Comment:
The VStack needs a background color because contents can be seen below it.


![](https://user-images.githubusercontent.com/5533851/275449302-9202bcfc-e9dd-4d56-8437-72cfd50a2bbd.png)

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10946/commits/4c7dbbd803153f7224898aa29c9c702de96a3283 , thanks!